staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -666,7 +666,7 @@ const (
 	// This label is used to indicate that this pod is the target of a migration job.
 	MigrationJobLabel string = "kubevirt.io/migrationJobUID"
 	// This label indicates the migration name that a PDB is protecting.
-	MigrationNameLabel string = "kubevirt.io/migrationName"
+	MigrationNameLabel string = "kubevirt.io/vmimName"
```

Comment:
Ignore my previous comment about this const. Still, I think we could have something like a `deprecated` old label and a new one.

Reply:
Yes maybe, I deleted it just to make sure no one is tempted to use it since it was only used internally, I think that's a stronger statement than having it with a 'deprecated' in the name (or in a comment), wdyt?